gyms.py

```diff
@@ -8,6 +8,7 @@
 import config
 import db
 import utils
+from decimal import *
```

Comment:
Ugh! Never ever `import *`! More so if you're only going to use `Decimal` from that module :crying_cat_face: